templates/tfengine/components/audit/main.tf

```diff
@@ -23,14 +23,12 @@ terraform {
 }
 
 {{- $parent_field := "org_id"}}
-{{- $parent_var := "var.org_id"}}
+{{- $parent_var := printf "\"%s\"" .parent_id}}
 {{- if eq .parent_type "folder"}}
 {{- $parent_field = "folder"}}
-{{- $parent_var = "var.folder_id"}}
+{{- $parent_var = printf "\"folders/%s\"" .parent_id}}
```

Comment:
Before we were using the folder id directly and now it's folders/. Is that OK for all resources within this module?

Reply:
yes

https://www.terraform.io/docs/providers/google/r/google_folder_iam_member.html
https://www.terraform.io/docs/providers/google/r/logging_folder_sink.html